yb-voyager/src/tgtdb/suites/oracleSuite.go

```diff
@@ -0,0 +1,206 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package tgtdbsuite
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"math"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var OraValueConverterSuite = map[string]ConverterFn{
+	"DATE": func(columnValue string, formatIfRequired bool) (string, error) {
+		// from oracle for DATE type debezium gives epoch milliseconds with type `io.debezium.time.Timestamp`
+		epochMilliSecs, err := strconv.ParseInt(columnValue, 10, 64) 
+		if err != nil {
+			return columnValue, fmt.Errorf("parsing epoch milliseconds: %v", err)
+		}
+		epochSecs := epochMilliSecs / 1000
+		parsedTime := time.Unix(epochSecs, 0).Local()
+		oracleDateFormat := "02-01-2006" //format: DD-MON-YYYY
+		formattedDate := parsedTime.Format(oracleDateFormat)
+		if err != nil {
+			return "", fmt.Errorf("parsing date: %v", err)
+		}
+		if formatIfRequired {
+			formattedDate = fmt.Sprintf("'%s'", formattedDate)
+		}
+		return formattedDate, nil
+	},
+	"io.debezium.time.Timestamp": func(columnValue string, formatIfRequired bool) (string, error) {
+		epochMilliSecs, err := strconv.ParseInt(columnValue, 10, 64)
+		if err != nil {
+			return columnValue, fmt.Errorf("parsing epoch milliseconds: %v", err)
+		}
+		epochSecs := epochMilliSecs / 1000
+		timestamp := time.Unix(epochSecs, 0).Local()
+		oracleTimestampFormat := "02-01-2006 03.04.05.000 PM" //format: DD-MM-YY HH.MI.SS.FFF PM
+		formattedTimestamp := timestamp.Format(oracleTimestampFormat)
+		if formatIfRequired {
+			formattedTimestamp = fmt.Sprintf("'%s'", formattedTimestamp)
+		}
+		return formattedTimestamp, nil
+	},
+	"io.debezium.time.MicroTimestamp": func(columnValue string, formatIfRequired bool) (string, error) {
```

Comment:
@priyanshi-yb pls take into consideration here the fixes i made in https://github.com/yugabyte/yb-voyager/pull/1023

Reply:
Also, test timestamp fields while streaming.